### ISSUE TITLE
example configuration

### ISSUE THREAD

**Issue by velociwabbit** [marmelab/admin-on-rest#396]
Presently the webpack.config.js in example does not work properly. There are two lines that need to be changed as follows.
```

const path = require('path');

module.exports = {
    devtool: 'eval',
    entry: './app.js',
    output: {
        path: path.join(__dirname, 'dist'),
        filename: 'bundle.js',
        publicPath: '/static/',
    },
    module: {
        loaders: [
        //    { test: /\.js$/, exclude: /node_modules/, loader: 'babel' },  ====this is the old line
            { test: /\.js$/, exclude: /node_modules/, loader: 'babel-loader' },
        //    { test: /\.css$/, loader: 'style!css' }, ==== this is the old line
            { test: /\.css$/, loader: 'style-loader!css-loader' },
       ],
    },
    resolve: {
        alias: {
            'admin-on-rest': path.join(__dirname, '..', 'src'),
        },
    },
};
```

**Comment by fzaninotto** [comment]
well, it does work for us!

@jpetitcolas do you know why our conf doesn't work for @velociwabbit ?

**Comment by jpetitcolas** [comment]
@velociwabbit: Which version of Webpack are you using? 

Indeed, Webpack team removed the possibility to omit the `-loader` suffix in 2.0 version (perhaps before?). I am :+1: for a PR adding them back yet, as this is the way they are meant to be in the future. Can you open a PR?

**Comment by velociwabbit** [comment]
i am on 2.2.1  of webpack.

will gladly open a PR

**Comment by jpetitcolas** [comment]
Fixed by #409.
